enclave-runtime/src/global_components.rs

```diff
@@ -91,3 +96,10 @@ pub static GLOBAL_PARENTCHAIN_IMPORT_DISPATCHER_COMPONENT: ComponentContainer<
 pub static GLOBAL_SIDECHAIN_BLOCK_IMPORTER_COMPONENT: ComponentContainer<
 	EnclaveSidechainBlockImporter,
 > = ComponentContainer::new();
+
+pub static GLOBAL_RPC_AUTHOR_COMPONENT: ComponentContainer<EnclaveRpcAuthor> =
+	ComponentContainer::new();
```

Comment:
Initializing this component was done in the sidechain crate until now - but since we have to pass in the o-call API now too, I decided to move it here into the `enclave-runtime` with the rest of all the global components.